webrtc.html

```diff
@@ -5152,6 +5224,45 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       on an <code>RTCRtpReceiver</code> are modified, a negotiation is
       triggered to signal the changes regarding what the application wants to
       receive to the other side.</p>
+      <p>To <dfn>create an RTCRtpReceiver</dfn> with kind, <var>kind</var>, and
+      optionally an id string, <var>id</var>, run the following steps:</p>
+      <ol>
+        <li>
+          <p>Let <var>sender</var> be a new <code><a>RTCRtpSender</a></code>
+          object.</p>
+        </li>
+        <li>
+          <p>Let <var>track</var> be a new <code><a>MediaStreamTrack</a></code>
+          object [[!GETUSERMEDIA]].</p>
+        </li>
+        <li>
+          <p>Initialize <var>track.kind</var> to <var>kind</var>.
```

Comment:
Yes, I think I like "Initialize" better than "set" on a newly created object, but we should be consistent.